tests/asr_benchmark_test.py

```
@@ -0,0 +1,280 @@
+
+        # Save and print results
+        save_benchmark_results(benchmark_results_dir, 'single_transcription', results_data)
+        print_performance_summary(
+            'Single Transcription Benchmark', gpu_info, audio_duration, num_runs, stats, rtf_metrics
+        )
+
+        # Performance assertion
+        assert stats['average_time'] < 30.0, f'Transcription too slow: {stats["average_time"]:.2f}s'
+
+    @pytest.mark.benchmark
+    def test_batch_transcription_performance(self, asr_model_warmed, test_audio_files, benchmark_results_dir):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_batch_transcription_performance`

Add return type annotation: `None`

(ANN201)